README.md

```diff
@@ -79,6 +89,21 @@ KeyFactory algorithms:
 * RSA
 * ACCP's SecureRandom uses AWS-LC's DRBG implementation.
 
+# Notes on ACCP-FIPS
+ACCP-FIPS is a variation of ACCP which is uses AWS-LC built in FIPS-mode.
```

Comment:
"which is uses" --> "which uses"
I acknowledge that I caused this typo as it was in my original suggested phrasing too.

Reply:
done